### PR TITLE
 #095/userContests/contestDetails

### LINKED ISSUE #95: UI | Konto użytkownika - contestDetails
image

### PR DESCRIPTION
**Pull request by Smyku6**
closes #95 
address for testing: add `/contestDetails` to our main address

### CODE REVIEW

File: src/Styles/fontsStyledComponents.js [hunk 1]
```diff
@@ -47,13 +47,13 @@ const FONTS = {
...
   line-height: 0.75rem;
-  letter-spacing: 0.009375rem;`,
+  letter-spacing: 0.09375rem;`,
 
   label_semibold: `font-size: 0.75rem;
   font-weight: 600;
   font-style: normal;
   line-height: 0.75rem;
-  letter-spacing: 0.009375rem;`,
+  letter-spacing: 0.09375rem;`,
```
Comment (Smyku6) [inline src/Styles/fontsStyledComponents.js:56]: we have error here. The right value is 0.09375rem not 0.009375

**Comment by Smyku6** [comment]
gooboi

File: src/Consts/ContestDetailsCONSTS.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export const toBeAnnounced = 'TBA';
+
```
Comment (MonikaKrella) [inline src/Consts/ContestDetailsCONSTS.js:2]: maybe "wkrótce" not TBA? i think polish user won't understand what means TBA

**Review by MonikaKrella — CHANGES_REQUESTED** [review]
image
it works almost ^ it is like that on ypur computer?
please merge latest development to your branch
but impressive!

File: src/Consts/ContestDetailsCONSTS.js (hunk 2 above)
Comment (Smyku6) [reply src/Consts/ContestDetailsCONSTS.js:2]: > image it works almost ^ it is like that on ypur computer? please merge latest development to your branch but impressive!

here for test I render two Components. First is with good_data and second i with no_data (this TBA things).

**Review by MonikaKrella — APPROVED** [review]
goood work!

**Pull request merged**